Fix filter for getting product exports

acris-cp opened the pull request:
### 1. Why is this change necessary?
Comparing the sales channel id for product export should be point on the selected storefront sales channel id instead.

### 2. What does this change do, exactly?
The current filter on the salesChannelId is not useful.

### 3. Describe each step to reproduce the issue or behaviour.


### 4. Please link to the relevant issues (if any).


### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

King-of-Babylon commented:
Thank you @hgiesenow for your PR.
This has already been implemented in Shopware 5.3.0 and therefore I will close this PR.

acris-cp committed:
Fix filter for getting product exports

Comparing the sales channel id for product export should be point on the selected storefront sales channel id instead.

PaddyS commented:
Hey @acris-cp,

thank you for your contribution! 💙 
We've just merged your changes! :-)
